src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -75,6 +75,17 @@ const QuizInner = () => {
   const isCorrect = grade === 1;
 
   const handleNextQuestionClick = () => {
+    track.questionAttemptSubmitted({
```

Comment:
This is in the wrong place. It should be triggered when you press the check button

Reply:
ok i thought this would be better than putting it in the quiz engine provider

Reply:
that was easy to fix